cmd/kubeadm/app/phases/upgrade/compute_test.go

```diff
@@ -892,7 +892,7 @@ func TestGetBranchFromVersion(t *testing.T) {
 			expectedVersion: "1.11",
 		},
 		{
-			version:         "1.12.5",
+			version:         "1.12.7",
```

Comment:
I think this is incorrect.

Reply:
It's my wrong. Thanks.

Reply:
@cblecker After I checked the code, the changes in this place didn't actually have any real impact.
It's just checking, and this part of the change doesn't change the final result.

Reply:
Do I need to modify it back?

Reply:
@tao12345666333 please remove this change.
1.12.5 here did not mean go version.

Reply:
@neolit123 Thanks. Addressed.